.github/workflows/build.yml

```diff
@@ -1,20 +0,0 @@
-name: build
```

Comment:
let's also add the [validate-generated-files](https://github.com/ConduitIO/conduit/blob/main/.github/workflows/validate-generated-files.yml) workflow plz, since we have paramgen and mockgen used